search/query/query_string.y

```diff
@@ -127,7 +127,12 @@ tSTRING tCOLON tSTRING tTILDE {
 tNUMBER {
 	str := $1
 	logDebugGrammar("STRING - %s", str)
-	q := NewMatchQuery(str)
+	q1 := NewMatchQuery(str)
+	val, _ := strconv.ParseFloat($1, 64)
```

Comment:
ya, we have this problem throughout the file though.  i'll add another commit with that, plus we need new category of tests for it